docker-compose_gpu.yaml

```diff
@@ -1,6 +1,22 @@
 version: '3.1'
+
 services:
+  ollama:
+    image: ollama/ollama
+    ports:
+      - "11434:11434"
+    restart: always
+    volumes:
+      - ollama_data:/root/.ollama
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: all
+              capabilities: [gpu]
   chromadb:
+    container_name: chromadb
```

Comment:
Let's not set container names in the docker, and allow the docker compose to manage the container names.